components/iam_app_user/README.md

```diff
@@ -0,0 +1,18 @@
+# IAM Application User
+
+This module creates an [IAM](https://aws.amazon.com/iam/) user and access key, which allows applications to access AWS resources.
+
+For all options, see the [variables](https://github.com/DoSomething/infrastructure/blob/main/components/iam_app_user/variables.tf) this module accepts & [outputs](https://github.com/DoSomething/infrastructure/blob/main/components/iam_app_user/outputs.tf) it generates.
+
+### Usage
+
+To create a new IAM user for the given application:
+
+```hcl
+module "log_archive" {
```

Comment:
Should this be `log_archive` or something else? I think this may be a copy-pasta mistake 🤔 ?

Reply:
Oof! Fixed this to be an example for `iam_app_user` in b3abf80.

Reply:
👍